.travis.yml

```diff
@@ -1,2 +1,4 @@
 language: java
 jdk: oraclejdk8
+before_install:
+  - chmod +x gradlew
```

Comment:
Is this really necessary? It would be better to commit it with the right flags if possible (presumably it works for mvnw).

Reply:
Is it really so important to not be explicit?